facets_mcp/tools/deploy_module.py

```
@@ -343,25 +376,33 @@ def get_deployment_logs(cluster_id: str, deployment_id: str) -> str:
+                    "cluster_id": cluster_id,
+                    "status": status,
+                    "log_count": len(formatted_logs),
+                    "logs": formatted_logs
+                }
             }, indent=2)
 
         except ApiException as e:
             if e.status == 404:
                 return json.dumps({
                     "success": False,
+                    "message": f"Failed to retrieve logs for deployment {deployment_id}. Verify the deployment ID.",
```

Pylint (3.3.7)

[convention] Line too long (116/100)

(C0301)